projects/ng-in-viewport/src/lib/directives/in-viewport.directive.ts

```diff
@@ -82,23 +82,31 @@ export class InViewportDirective implements AfterViewInit, OnDestroy {
     if (isPlatformBrowser(this.platformId)) {
       this.inViewportService.unregister(this.nativeElement, this.config);
     }
+
+    this.emit(undefined, true, false);
```

Comment:
I believe this line should be moved right behind line  `this.inViewportService.unregister(this.nativeElement, this.config);` and wrapped by the `if (isPlatformBrowser(this.platformId)) { }` ... no point in emitting if on SSR...